app/lib/shared/redis_cache.dart

```diff
@@ -232,6 +232,11 @@ class CachePatterns {
         encode: (PublisherPage list) => list.toJson(),
         decode: (data) => PublisherPage.fromJson(data as Map<String, dynamic>),
       ))['$userId'];
+
+  Entry<String> atomFeedXml() => _cache
```

Comment:
I'm guessing that caching is also a new thing :D

Reply:
Ah, yeah, slightly related :)